### PR TITLE
chore: Added ignore build step for next deployments

### PR DESCRIPTION
**Pull request by dackers86**
### Checklist

- I read the [Contributor Guide](../CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
  - [ ] Yes
- My change supports the following platforms;
  - [ ] `Android`
  - [ ] `iOS`
- My change includes tests;
  - [ ] `e2e` tests added or updated in `packages/\*\*/e2e`
  - [ ] `jest` tests added or updated in `packages/\*\*/__tests__`
- [ ] I have updated TypeScript types that are affected by my change.
- This is a breaking change;
  - [ ] Yes
  - [ ] No

### Test Plan

---

Think `react-native-firebase` is great? Please consider supporting the project with any of the below:

- 👉 Star this repo on GitHub ⭐️
- 👉 Follow [`React Native Firebase`](https://twitter.com/rnfirebase) and [`Invertase`](https://twitter.com/invertaseio) on Twitter

### CODE REVIEW

File: script.sh
Comment (mikehardy) [inline script.sh:15]:
```suggestion
    echo "✅ - Detected this is a build of @invertase/next - build can proceed"
```

File: script.sh
Comment (mikehardy) [inline script.sh:20]:
```suggestion
  echo "🛑 - This is not a build of @invertase/next build cancelled"
```

**Review by mikehardy — COMMENTED** [review]
I'm sure the idea in general is useful for your vNext work and I like both the idea and the implementation. So then I pictured a future frame of reference where it was maybe in the wee hours of the morning and I was trying to decipher why a build was (or was) not doing what I wanted, and proposed altered wording to give my future exhausted self more explicit breadcrumbs in the logs just in case. What do you think?

**Comment by dackers86** [comment]
> I'm sure the idea in general is useful for your vNext work and I like both the idea and the implementation. So then I pictured a future frame of reference where it was maybe in the wee hours of the morning and I was trying to decipher why a build was (or was) not doing what I wanted, and proposed altered wording to give my future exhausted self more explicit breadcrumbs in the logs just in case. What do you think?

Agreed! I really did think there used to be a different way of doing this. This should only be necessary while developing the new modules in parallel, once that branch is merged we can simply delete the file. The Vercel config for this is restricted to the preview deployment only! ie. `react-native-firebase-next`

**Review by mikehardy — APPROVED** [review]
If this is ready in the vercel config, this LGTM - hit that merge button :-)

**Comment by dackers86** [comment]
Setting updated.

image

**Comment by mikehardy** [comment]
:zap:

**Pull request merged**
